changelogs/fragments/1950-elb_application_lb-multiple-listener-certificates-for-alb.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- elb_application_lb - Add support for adding listener with multiple certificates during ALB creation (https://github.com/ansible-collections/amazon.aws/pull/1950).
```

Comment:
I would rather prefer stating that the change in on `module_utils/elbv2` and then mention that it brings benefit for the `elb_application_elb` module